Cargo.toml

```diff
@@ -117,7 +117,7 @@ lazy_static = "1.3.0"
 rlua = { git = "https://github.com/timberio/rlua" }
 num_cpus = "1.10.0"
 bytesize = "1.0.0"
-grok = "1.0.0"
+grok = "1.0.1"
```

Comment:
I think it should be `grok = "~1.0.1"` because by default Cargo pulls version 1.1.0.

Reply:
I've added these changes to the branch, as soon as the tests pass, I'll merge.